src/Client/app/(root)/my-exams/[id]/page.tsx

```
@@ -33,41 +33,67 @@ export default function Component() {
-        }
-        if (e.ctrlKey || e.altKey || e.metaKey) {
+          }
+          if ((e.ctrlKey || e.metaKey) && (e.key === 'c' || e.key === 'v' || e.key === 'x')) {
             e.preventDefault();
             e.stopPropagation();
             return;
-        }
-        let key = e.key;
-        if (key.length === 1 && /^[A-Za-z]$/.test(key)) {
+          }
+          if (e.key.startsWith('F') && parseInt(e.key.substring(1)) >= 1 && parseInt(e.key.substring(1)) <= 12) {
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)